file_test.go

```diff
@@ -42,13 +42,13 @@ func TestIssue11NoNoteSectionELF(t *testing.T) {
 	if err != nil {
 		panic("No go tool chain found: " + err.Error())
 	}
-	tmpdir, err := ioutil.TempDir("", "TestGORE-Issue11")
+	tmpdir, err := os.MkdirTemp("", "TestGORE-Issue11")
```

Comment:
I think the better change is to use the temp dir function provided by `t` since it will be removed automatically once the tests are done.